packages/react-components/lib/tasks/task-timeline.tsx

```diff
@@ -48,47 +47,99 @@ const StyledTimeLine = styled((props: TimeLinePropsWithRef) => <Timeline {...pro
   }),
 );
 
+function nestedEvents(eventsStates: { [key: string]: EventState } | undefined, child: number) {
+  if (eventsStates !== undefined) {
+    const deps = eventsStates[child] ? eventsStates[child].deps : [];
+    return (
+      <TreeItem
+        key={`event-${child}`}
+        nodeId={`parent-${eventsStates[child].name}`}
+        label={eventsStates[child].name}
+      >
+        {deps?.map((id) => {
+          if (eventsStates[id].deps?.length) {
+            nestedEvents(eventsStates, id);
+          } else {
+            return (
+              <TreeItem
+                key={eventsStates[child].name}
+                nodeId={`child-${id}`}
+                label={eventsStates[id].name}
+              ></TreeItem>
+            );
+          }
+        })}
+      </TreeItem>
+    );
+  }
+}
+
+function colorDot(eventsStates: { [key: string]: EventState } | undefined) {
+  let standby: Boolean = true;
```

Comment:
This causes lint error
```
Don't use `Boolean` as a type. Use boolean instead
```

---

Which lead me to realize that the linter is not ran on the CI :sweat: . You can run it manually with `npm run lint`.